llvm/lib/ObjectYAML/COFFYAML.cpp

```diff
@@ -183,6 +183,25 @@ void ScalarEnumerationTraits<COFF::RelocationTypeAMD64>::enumeration(
   ECase(IMAGE_REL_AMD64_SSPAN32);
 }
 
+void ScalarEnumerationTraits<COFF::RelocationTypesMips>::enumeration(
+    IO &IO, COFF::RelocationTypesMips &Value) {
+  ECase(IMAGE_REL_MIPS_ABSOLUTE);
```

Comment:
You probably don't need to test every case here, but I think your new test should include a couple of relocations, so that this code is exercised somehow by the test.